xwiki-platform-core/xwiki-platform-oldcore/src/main/java/com/xpn/xwiki/doc/XWikiDocument.java

```diff
@@ -635,6 +641,11 @@ private static LinkParser getLinkParser() {
 
     private RenderingContext renderingContext;
 
+    /**
+     * @see #getAuthors()
+     */
+    private DefaultDocumentAuthors authors;
```

Comment:
Now that DocumentAuthors is modifiable, it would IMO make more sense to set this field from the start and then always modify it.

Reply:
Done.